packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -189,8 +191,9 @@ export function QueueInfoColumn({
         <DisableQueueButton
           onClick={confirmDisable}
           data-cy="queue-disable-button"
+          disabled={queue?.isDisabled}
```

Comment:
It might be worth it to also disable the Check In button (for TAs) and the Join Queue button (for students) if the queue is disabled - it looks like those buttons are passed in via `TAQueue.tsx` and `StudentQueue.tsx`. We could put a tooltip on the button explaining that you can't check into disabled queues as well.

Reply:
apparently tooltips dont show up on disabled elements :(

Reply:
We have something like this somewhere I think? Like if you try to help a student when you aren't checked in

Reply:
Yeah [heres an example](https://github.com/sandboxnu/office-hours/pull/800/files/1cadd2f188d2bdb387dbf3d68be3f42a502638fa..8ec16ff01a89284e40000c993fb96dc39aeb6a73#diff-a76a51c3539f561d5b49ce8fdc4ea95961d162134f96dad909f3653acedab916R134) of a conditional tooltip on a disabled button